### PR TITLE
refactor(rxTimePicker): FRMW-764 migrate to Forms element

### PR DESCRIPTION
**Pull request by 100stacks**
JIRA: https://jira.rax.io/browse/FRMW-764

### LGTMs
- [x] Dev LGTM
- [x] Dev LGTM
- [x] Design LGTM

---
Refactor `rxTimePicker` into `Elements > Forms` element.

### CODE REVIEW

**Comment by 100stacks** [comment]
## `Elements > Forms > Time Picker` Demo Page

encoreui 2016-05-06 14-41-44

## API Docs

rxtimepicker 2016-05-06 13-41-42

elements 2016-05-06 14-18-32

**Comment by parlarjb** [comment]
LGTM

File: src/elements/Forms/docs/forms.docs.html [hunk 1]
```diff
@@ -424,6 +424,95 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
 </ul>
 <rx-example name="forms.disabled"></rx-example>
 
+<!-- BEGIN: rxTimePicker -->
+<div ng-controller="rxTimePickerCtrl">
+  <h2>
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: Please convert to `<h3>`

File: src/elements/Forms/docs/forms.docs.html [hunk 2]
```diff
@@ -424,6 +424,95 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
...
+  </h2>
+  <rx-notification type="info">
+    <p>
+      This element is designed to be used in conjunction with other picker
+      elements to compose a valid ISO 8601 DateTime string in the format of
+      <code>YYYY-MM-DDTHH:mmZ</code>.
+    </p>
+  </rx-notification>
+
+  <h2>Notice</h2>
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: I'd remove this heading. The notification and list should be enough to describe the element.

File: src/elements/Forms/docs/forms.docs.html [hunk 3]
```diff
@@ -424,6 +424,95 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
...
+        <li><code>mm</code> is the minutes from 00 to 59</li>
+        <li><code>Z</code> is the UTC offset that matches <code>[-+]\d{2}:\d{2}</code></li>
+      </ul>
+    </li>
+    <li>
+      This element will never generate anything other than a String.
+    </li>
+  </ul>
+
+  <h2>Examples of Usage</h2>
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: I'd remove this heading.

File: src/elements/Forms/docs/forms.docs.html [hunk 4]
```diff
@@ -424,6 +424,95 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
...
+  <h3>Simple Example</h3>
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: Please change to `<h4>`

File: src/elements/Forms/docs/forms.docs.html [hunk 5]
```diff
@@ -424,6 +424,95 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
...
+  <rx-example name="rxTimePicker.simple"></rx-example>
+
+  <h2>Time Picker States</h2>
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: Please change to `<h4>`

File: src/elements/Forms/docs/forms.docs.html [hunk 6]
```diff
@@ -424,6 +424,95 @@ <h2 class="clear"><rx-permalink>Directives</rx-permalink></h2>
 </ul>
 <rx-example name="forms.disabled"></rx-example>
 
+<!-- BEGIN: rxTimePicker -->
+<div ng-controller="rxTimePickerCtrl">
```
Comment (CITguy) [inline src/elements/Forms/docs/forms.docs.html]: Wrapper `<div>` unnecessary. You should be able to move the `ng-controller` attribute to the state table.

```html
<table ng-controller="rxTimePickerCtrl">
```

**Comment by 100stacks** [comment]
@CITguy, feedback incorporated.

**Comment by glynnis** [comment]
LGTM

**Pull request merged**